src/plays/sierpinski-triangle/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+import P5 from "p5";
+
+export const generateRandomColor = (p: P5): P5.Color => {
+  return p.color(p.random(0, 250), p.random(0, 250), p.random(0, 250));
+};
```

Comment:
```suggestion
export const generateRandomColor = (p: P5): P5.Color => p.color(p.random(0, 250), p.random(0, 250), p.random(0, 250));

```

Reply:
this one do intensionally for readablity, I find it more convinient

Reply:
@bhtibrewal , Here we are just returning the value so we should directly do that, it's the real power of the arrow function.